README.md

```diff
@@ -1,2 +1,185 @@
+
+![Screenshot (40)](https://github.com/davidolanrewaju/WeatherApp/assets/63909901/7482e87d-4a66-4f5b-8aeb-1eb40e7415f5)
+
+![Screenshot (42)](https://github.com/davidolanrewaju/WeatherApp/assets/63909901/5a27dedd-4bb1-4eea-8c42-a9f3519e6162)
+
+![Screenshot (43)](https://github.com/davidolanrewaju/WeatherApp/assets/63909901/0e174237-7dbf-4b2f-8b5a-208221532a98)
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+<details>
+  <summary>Frontend</summary>
+  <ul>
+    <li>CSS</li>
+    <li>Linters</li>
+    <li>GitHub Flow</li>
+    <li>Git Flow</li>
+    <li>Javascript</li>
+    <li>React</li>
+    <li>Redux Toolkit</li>
+    <li>React Router DOM </li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- [x] Displays a list of cities
+- [x] Displays weather information and forecast for a selected city.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+### 🚀 Live Demo <a name="live-demo"></a>
+- [Live Demo Link](https://weatherapp-io.onrender.com/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+<ul>
+  <li>GitHub Account</li>
+  <li>Git installed on your system</li>
+  <li>Install all dependencies</li>
+  <li>Code Editor</li>
+</ul>
+
+### Setup
+
+Clone this repository to your desired folder, write the following commands in git bash 
+
+```sh
+  cd my-folder
+  git clone https://github.com/davidolanrewaju/WeatherApp.git
+```
+
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  npm init -y
+  npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+  npm install --save-dev eslint@7.x eslint-config-airbnb-base@14.x eslint-plugin-import@2.x babel-eslint@10.x
+  npx create-react-app
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+```sh
+  npx stylelint "**/*.{css,scss}"
+  npx eslint "**/*.{js,jsx}"
+  npm run build
+```
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Olanrewaju David**
+- GitHub: [@davidolanrewaju](https://github.com/davidolanrewaju)
+- Twitter: [@ola_dayve](https://twitter.com/ola_dayve)
+- LinkedIn: [David Olanrewaju](https://www.linkedin.com/in/david-olanrewaju)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- VIDEO SUMMARY -->
+## 🎥 Video Summary <a name="video-summary"></a>
+>This is a video summary on the aim of this project and the challenges encountered.</br>
+Click [this link](https://www.loom.com/share/f02739603fca4d7c83ee79ed7d6c995a) to get a view of the project
+
```

Comment:
- [ ] Dear @davidolanrewaju, You have don great on the Implementation, I also, I like your video presentation. But, Please remember the video is expected to contain a highlight about a piece of the code that you find interesting or you are proud of making. Please, consider redoing the video with the `piece of the code that you find interesting` alongside the objective description and demo of the project.